pkg/scheduler/schedule_one.go

```diff
@@ -842,9 +841,13 @@ func (sched *Scheduler) handleSchedulingFailure(ctx context.Context, fwk framewo
 	}
 
 	pod := podInfo.Pod
-	var errMsg string
-	if err != nil {
-		errMsg = err.Error()
+	var (
+		err    error
+		errMsg string
+	)
+	if !status.IsSuccess() {
```

Comment:
status is always unsuccess here, no?

Reply:
Yes you're right.It is err that can be nil.